cardano-cli/src/Cardano/CLI/Shelley/Commands.hs

```diff
@@ -211,6 +305,19 @@ data GenesisCmd
   | GenesisHashFile GenesisFile
   deriving (Eq, Show)
 
+renderGenesisCmd :: GenesisCmd -> Text
+renderGenesisCmd cmd =
+  case cmd of
+    GenesisCreate {} -> "genesis create"
+    GenesisKeyGenGenesis {} -> "genesis key-gen-genesis"
+    GenesisKeyGenDelegate {} -> "genesis key-gen-delegate"
+    GenesisKeyGenUTxO {} -> "genesis key-gen-utxo"
+    GenesisCmdKeyHash {} -> "genesis key-hash"
+    GenesisVerKey {} -> "genesis get-ver-key"
+    GenesisTxIn {} -> "genesis initial-txin"
+    GenesisAddr {} -> "genesis initial-addr"
+    GenesisHashFile {} -> "genesis hash"
```

Comment:
Actually, I'm curious how in this case constructor name and the RHS text don't match.

Reply:
We can change the `Strings` to match the constructor names but the `Strings` are meant to match the `cardano-cli` commands

Reply:
I was thinking to change the constructor names to match the strings.

Reply:
Sorry, I do not understand, what is the question?

Reply:
> I was thinking to change the constructor names to match the strings, although not necessarily in this PR.
Yeah I don't mind doing this in a follow up PR.

Reply:
> Sorry, I do not understand, what is the question?

For example in the code:

```
GenesisHashFile {} -> "genesis hash"
```

The words on the LHS are not the same as the RHS, but it seemed like in most cases it is and perhaps would be good if it consistently was the case.